templates/partials/devel_build.dockerfile.j2

```diff
@@ -10,7 +10,6 @@ RUN cd ${RAPIDS_DIR}/{{ lib.name }} && \
   {% if lib.name == "cudf" %}
   ./build.sh -l && \
   cd cpp/build && \
-  make -j${PARALLEL_LEVEL} build_tests_nvstrings && \
   make -j${PARALLEL_LEVEL} build_tests_cudf
```

Comment:
I'd replace this with ./build.sh tests so they don't break us in the future by changing target names, etc.